### ISSUE TITLE
Dialog to connect wallet is shown behind other dialogs

### ISSUE THREAD

**Issue by ebma** [pendulum-chain/portal#497]
### Describe the bug

When clicking on the 'Connect to wallet' button in the backstop pool 'Withdraw' dialog, the dialog for connecting the wallet is shown behind the current one and I cannot interact with it.  

### Reproduction

1. On 'Foucoco', without being connected to an account
2. Click on 'Forex AMM' > 'Backstop Pool' in the sidebar
3. Click on 'Withdraw'
4. Click on 'Connect to wallet'

### Expected behavior

The dialog to connect to a wallet account should be shown on top.

### Screenshots

<img width="953" alt="image" src="https://github.com/pendulum-chain/portal/assets/6690623/a36673cd-3ffe-4f4f-a09d-9767210639aa">

**Comment by ebma** [comment]
@prayagd I created a new bug ticket.

**Comment by ebma** [comment]
I tried some things with around changing the `z-index` field but I didn't manage to show the wallet selection dialog on top of the other. I think we should postpone fixing this to after the revamped wallet selection is available.

**Comment by ebma** [comment]
I tested this and this will be fixed once https://github.com/pendulum-chain/portal/pull/498 is merged.

**Comment by prayagd** [comment]
Moving this ticket to development
